bundles/org.openhab.binding.omnilink/src/main/resources/OH-INF/thing/audio-zone.xml

```diff
@@ -13,10 +13,10 @@
 		<description>An audio zone configured in the controller.</description>
 		<channels>
 			<channel id="zone_power" typeId="audio_zone_power"/>
-			<channel id="zone_mute" typeId="audio_zone_mute"/>
-			<channel id="zone_volume" typeId="audio_zone_volume"/>
+			<channel id="zone_mute" typeId="system.mute"/>
```

Comment:
It's a good idea to use system channel types when possible, but you will need to add own label and description now since they are no longer taken from the now deleted custom channel type.

Reply:
How would I add just a description and nothing else? I may being missing something but I don't see an example in the docs to reference.

Reply:
See #11668 for inspiration as this was a similar case.

Reply:
Fixed